LICENSE.txt

```diff
@@ -0,0 +1,202 @@
+
+                                 Apache License
```

Comment:
Thi service is Apache licensed or `Traceable Community License`?

Reply:
It's going to be a mix - span-normalizer is under Apache, but hypertrace-trace-enricher is Community.

Reply:
Let me go with the community version at the root level, and the individual jobs will have accordingly.